--- origsrc/ibus-anthy-1.5.7/engine/python2/Makefile.am	2015-07-23 18:49:55.910620100 -0500
+++ src/ibus-anthy-1.5.7/engine/python2/Makefile.am	2015-07-23 18:50:34.677542900 -0500
@@ -79,8 +79,8 @@ engine_DATA = default.xml
 enginedir = $(pkgdatadir)/engine
 
 %.xml: %.xml.in
-	sed -e "s|\@libexecdir\@|$(libexecdir)|g" \
-	    -e "s|\@pkgdatadir\@|$(pkgdatadir)|g" \
+	sed -e "s|\@LIBEXECDIR\@|$(libexecdir)|g" \
+	    -e "s|\@PKGDATADIR\@|$(pkgdatadir)|g" \
 	    -e "s|_description|description|g" \
 	$< > $@
 
--- origsrc/ibus-anthy-1.5.7/engine/python2/anthy.xml.in.in	2015-07-16 04:04:18.000000000 -0500
+++ src/ibus-anthy-1.5.7/engine/python2/anthy.xml.in.in	2015-07-23 18:51:23.942298700 -0500
@@ -3,7 +3,7 @@
 <component>
 	<name>org.freedesktop.IBus.Anthy</name>
 	<description>Anthy Component</description>
-	<exec>@libexecdir@/ibus-engine-anthy --ibus</exec>
+	<exec>@LIBEXECDIR@/ibus-engine-anthy --ibus</exec>
 	<version>@PACKAGE_VERSION@</version>
 	<author>Peng Huang &lt;shawn.p.huang@gmail.com&gt;</author>
 	<license>GPL</license>
@@ -13,7 +13,7 @@
 	<!-- for engines -->
 	<observed-paths>
 		<path>~/.config/ibus-anthy/engines.xml</path>
-		<path>@pkgdatadir@/engine/default.xml</path>
+		<path>@PKGDATADIR@/engine/default.xml</path>
 	</observed-paths>
-	<engines exec="@libexecdir@/ibus-engine-anthy --xml" />
+	<engines exec="@LIBEXECDIR@/ibus-engine-anthy --xml" />
 </component>
--- origsrc/ibus-anthy-1.5.7/engine/python3/Makefile.am	2015-07-23 18:49:55.913620500 -0500
+++ src/ibus-anthy-1.5.7/engine/python3/Makefile.am	2015-07-23 18:50:49.476922200 -0500
@@ -79,8 +79,8 @@ engine_DATA = default.xml
 enginedir = $(pkgdatadir)/engine
 
 %.xml: %.xml.in
-	sed -e "s|\@libexecdir\@|$(libexecdir)|g" \
-	    -e "s|\@pkgdatadir\@|$(pkgdatadir)|g" \
+	sed -e "s|\@LIBEXECDIR\@|$(libexecdir)|g" \
+	    -e "s|\@PKGDATADIR\@|$(pkgdatadir)|g" \
 	    -e "s|_description|description|g" \
 	$< > $@
 
--- origsrc/ibus-anthy-1.5.7/engine/python3/anthy.xml.in.in	2015-07-16 04:04:18.000000000 -0500
+++ src/ibus-anthy-1.5.7/engine/python3/anthy.xml.in.in	2015-07-23 18:51:30.030571800 -0500
@@ -3,7 +3,7 @@
 <component>
 	<name>org.freedesktop.IBus.Anthy</name>
 	<description>Anthy Component</description>
-	<exec>@libexecdir@/ibus-engine-anthy --ibus</exec>
+	<exec>@LIBEXECDIR@/ibus-engine-anthy --ibus</exec>
 	<version>@PACKAGE_VERSION@</version>
 	<author>Peng Huang &lt;shawn.p.huang@gmail.com&gt;</author>
 	<license>GPL</license>
@@ -13,7 +13,7 @@
 	<!-- for engines -->
 	<observed-paths>
 		<path>~/.config/ibus-anthy/engines.xml</path>
-		<path>@pkgdatadir@/engine/default.xml</path>
+		<path>@PKGDATADIR@/engine/default.xml</path>
 	</observed-paths>
-	<engines exec="@libexecdir@/ibus-engine-anthy --xml" />
+	<engines exec="@LIBEXECDIR@/ibus-engine-anthy --xml" />
 </component>
